elections/2022-TOC/election_desc.md

```diff
@@ -0,0 +1,27 @@
+# 2022 Voters Guide - Knative TOC Election
+
+The role of this election is to fill the four (4) seats due for election this year on the [Knative Technical Oversight Committee](https://github.com/knative/community/blob/master/TECH-OVERSIGHT-COMMITTEE.md). The elected members will serve two (2) year terms.
+
+This election will shape the future of Knative as a community and project.
+While WGs help shape the individual direction of the project, the
+TOC Charter covers the technical direction, health of the project
+and community as a whole.
+
+This is a brief summary of the election; for more details, such as how to become a candidate, [see the README](https://github.com/knative/community/blob/main/elections/2022-TOC/README.md).
+
+## Eligibility
+
+All Knative contributors with 50 or more contributions over the last year are eligible to vote.  You will see either ELIGIBLE or NOT ELIGIBLE at the top of this screen.  If you are not eligible, but should be, please [file an exception request](https://test.elekto.io/app/elections/2022-TOC/exception) or contact elections@knative.team.
```

Comment:
Done! I've added the link, but I added the complete link, instead of bit.ly, so that people can hover over it and see that it's a devstats link before they click it.